be/test/CMakeLists.txt

```diff
@@ -271,6 +267,11 @@ if ("${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86" OR "${CMAKE_BUILD_TARGET_ARCH}"
         set_source_files_properties(./formats/json/nullable_column_test.cpp PROPERTIES COMPILE_FLAGS -mno-avx2)
 endif()
 
+if (WITH_AWS)
```

Comment:
Please ensure the following tests always be compiled, but skip running them when there is no available S3 environment.

Reply:
It's possible but I don't know any plausible solution now. I have to do some search.

Reply:
Run tests with the flag `--gtest-filter=-*S3*` should works.